website/docs/contributing/release.md

```diff
@@ -0,0 +1,372 @@
+svn co https://dist.apache.org/repos/dist/dev/incubator/opendal opendal-dist-dev
+```
+
+Then, upload the artifacts:
+
+> The `${release_version}` here should be like `0.36.0-rc1`
+
+```shell
+cd opendal-dist-dev
+# create a directory named by version
+mkdir ${release_version}
+# copy source code and signature package to the versioned directory
+cp ${repo_dir}/dist/* ${release_version}/
+# check svn status
+svn status
+# add to svn
+svn add ${release_version}
+# check svn status
+svn status
+# commit to SVN remote server
+svn commit -m "Prepare for ${release_version}"
+```
+
+Visit <https://dist.apache.org/repos/dist/dev/incubator/opendal/> to make sure the artifacts is uploaded correctly.
+
+## Voting
+
+As an incubating project, OpenDAL requires votes from both the OpenDAL Community and Incubator Community.
+
+> opendal_version: the version for opendal, like `0.36.0`
+> release_version: the version for voting, like `0.36.0-rc1`
+
+### OpenDAL Community Vote
+
+OpenDAL Community Vote should send email to: <dev@opendal.apache.org>:
+
+Title:
+
+```
+[VOTE] Release Apache OpenDAL(incubating) ${release_version} - OpenDAL Vote Round 1
+```
+
+Content:
+
+```
+Hello, Apache OpenDAL(incubating) Community,
+
+This is a call for a vote to release Apache OpenDAL(incubating) version ${opendal_version}.
+
+The tag to be voted on is ${opendal_version}.
+
+The release candidate:
+
+https://dist.apache.org/repos/dist/dev/incubator/opendal/${release_version}/
+
+Keys to verify the release candidate:
+
+https://downloads.apache.org/incubator/opendal/KEYS
+
+Git branch for the release:
+
+https://github.com/apache/incubator-opendal/tree/release-${release_version}
+
+Please download, verify, and test.
+
+The VOTE will pass after got 3 binding approve.
+
+[ ] +1 approve
+[ ] +0 no opinion
+[ ] -1 disapprove with the reason
+
+To learn more about apache opendal, please see https://opendal.apache.org/
+
+Checklist for reference:
+
+[ ] Download links are valid.
+[ ] Checksums and signatures.
+[ ] LICENSE/NOTICE files exist
+[ ] No unexpected binary files
+[ ] All source files have ASF headers
+[ ] Can compile from source
+
+More detailed checklist please refer to:
+https://github.com/apache/incubator-opendal/tree/main/scripts
+
+To compile from source, please refer to:
+https://github.com/apache/incubator-opendal/blob/main/CONTRIBUTING.md
+
+Thanks
+
+${name}
+```
+
+Example: <https://lists.apache.org/thread/c211gqq2yl15jbxqk4rcnq1bdqltjm5l>
+
+After at least 3 +1 binding vote (from OpenDAL Podling PMC member and committers) and no veto, claim the vote result:
+
+Title:
+
+```
+[Result][VOTE] Release Apache OpenDAL(incubating) ${release_version} - OpenDAL Vote Round 1
+```
+
+Content:
+
+```
+Hello, Apache OpenDAL(incubating) Community,
+
+The vote to release Apache OpenDAL(Incubating) ${release_version} has passed.
+
+The vote PASSED with 3 binding +1 and 0 -1 vote:
+
+Binding votes:
+
+- Xuanwo (binding)
+- Sundyli (binding)
+- Chojan Shang (binding)
```

Comment:
I think this is not a part of the template.

Reply:
fixed.